source/io/module_httpClient.js

```diff
@@ -71,45 +70,7 @@
         // None
 
         // Static Public Functions
-        // https://fetch.spec.whatwg.org/#forbidden-header-name
-        var forbiddenHeaderNames = [
```

Comment:
I wouldn't leave the names, it is liable to get outdated. The link to the spec is probably sufficient.